### PR TITLE
Fix ADCM schema for action masking

### PR DESCRIPTION
**Pull request by nagarin**

### CODE REVIEW

File: python/cm/models.py [hunk 1]
```diff
@@ -876,6 +876,12 @@ def get_id_chain(self, target_ids: dict) -> dict:
 
     def allowed(self, obj: ADCMEntity) -> bool:
         """Check if action is allowed to be run on object"""
+        if self.state_unavailable == 'any':
+            return False
+
+        if self.multi_state_unavailable == 'any':
+            return False
+
```
Comment (dgusakov) [inline python/cm/models.py:884]: So we drop support of ANy here?
@acmnu Are you ok with it?

**Review by Sealwing — APPROVED** [review]

File: python/cm/models.py (hunk 1 above)
Comment (nagarin) [reply python/cm/models.py:884]: Otherwise, I added support for 'any' in unavailable, as it was discussed.

Comment (dgusakov) [reply python/cm/models.py:884]: Ok. Thx for clarifying

**Comment by acmnu** [comment]
@Sealwing Is it done?

**Review by Sealwing — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**